interface/src/app/mlmodel/selector.tpl.html

```diff
@@ -1,8 +1,8 @@
 <div class="alert alert-danger" role="alert" ng-if="$ctrl.errorMessage">
   {{$ctrl.errorMessage}}
 </div>
-<select
-  ng-options="option as option.title for option in $ctrl.availableMlModels"
-  ng-model="$ctrl.selectedMlModel">
-  <option value="">--- Please select a machine learning model ---</option>
+<select name="selectMlModel"
+  ng-options="opt.title for opt in $ctrl.availableMlModels track by opt.id"
```

Comment:
I was following the documentation here: https://docs.angularjs.org/api/ng/directive/select#using-select-with-ngoptions-and-setting-a-default-value

I can try again without the `track by` part. Maybe that wasn't the change that actually made it work? I think it's a good idea to use `.id` for tracking, though. It just makes more sense to me than using the `.title`.